tests/integration/test_mysql_protocol/test.py

```diff
@@ -416,6 +422,15 @@ def test_golang_client(server_address, golang_container):
     assert stdout == reference
 
 
+def test_dotnet_client(server_address, dotnet_container):
+    code, (stdout, stderr) = dotnet_container.exec_run(
+        'dotnet run --host {host} --port {port} --user default --password 123 --database default'
+        .format(host=server_address, port=server_port), demux=True)
```

Comment:
Why do you add it to mysql_protocol test? Which protocol is used? server_port is a native one.

Reply:
I've used golang client as base example, it's in same file. What is the variable with http port?

Reply:
`test_mysql_protocol` has `test_golang_client` to test MySQL compatibility protocol of ClickHouse by using golang MySQL driver.

But your .NET driver does not use MySQL compatibility protocol. So, better to make a separate test directory for it.